pkg/collector/corechecks/cluster/orchestrator/processorstest/context.go

```diff
@@ -89,6 +95,11 @@ func (pc *ProcessorContext) GetKind() string {
 	return pc.Kind
 }
 
+//nolint:revive
```

Comment:
As a curiosity, why are we skipping linting rules here instead of adding a comment to the function? 

No need to update it in this PR as the other functions are doing the same but something to consider in future changes.